### PR TITLE
New CI workflow

### PR DESCRIPTION
**Pull request by pbrudnick**

### CODE REVIEW

File: .github/workflows/erlang.yml [hunk 1]
```diff
@@ -10,17 +10,21 @@ jobs:
 
     strategy:
       matrix:
-        otp: [21.3, 22.3, 23.0.4, 23.2.7.0, 24.0]
-    container:
-      image: erlang:${{ matrix.otp }}-alpine
+        combo:
+          - otp: ['21.3', '22.3', '23.3']
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/erlang.yml:14]: I also wished it was this intelligent, but alas you have to separate all the different combos, it seems. 😢

**Review by elbrujohalcon — APPROVED** [review]

File: .github/workflows/erlang.yml [hunk 2]
```diff
@@ -10,17 +10,21 @@ jobs:
 
     strategy:
       matrix:
-        otp: [21.3, 22.3, 23.0.4, 23.2.7.0, 24.0]
-    container:
-      image: erlang:${{ matrix.otp }}-alpine
+        otp: ['21.3', '22.3', '23.3']
+        rebar: ['3.15.2']
+        include:
+        - otp: '24.0'
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/erlang.yml:16]: 3.16.1 goes all the way down to 22, so the exception to the rule is actually 21.3/3.15.2 :)

File: .github/workflows/erlang.yml [hunk 3]
```diff
@@ -10,17 +10,21 @@ jobs:
...
+          rebar: '3.16.1'
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/erlang.yml:17]:
```suggestion
        otp: ['22.3', '23.3', '24.0']
        rebar: ['3.16.1']
        include:
        - otp: '21.3'
          rebar: '3.15.2'
```

Comment (paulo-ferraz-oliveira) [reply .github/workflows/erlang.yml:17]: This way, when the time to stop supporting 21.3 comes you only have to delete stuff 😄

**Pull request merged**